components/registry-facade/pkg/registry/layersource.go

```diff
@@ -368,6 +368,9 @@ func (cs CompositeLayerSource) GetBlob(ctx context.Context, spec *api.ImageSpec,
 // RefSource extracts an image reference from an image spec
 type RefSource func(*api.ImageSpec) (ref string, err error)
 
+// ResfSource extracts an image list reference from an image spec
+type RefsSource func(*api.ImageSpec) (refs []string, err error)
```

Comment:
Better refactor `RefSource` to return `[]string` instead of introducing another one on the side